packages/firebase_crashlytics/CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
+## 0.1.1
+
+* Added additional exception information from the Flutter framework to the reports.
+* Reformatted debug printing of exceptions to be human-readable.
+* Updated `README.md` to include both the breaking change from `0.1.0` and the newly added
+  `recordError` function in the setup section.
+* Made the `_recordError` function safe for missing stack traces (`null`).
```

Comment:
```suggestion
* Passing `null` stack traces is now supported.
```

Since `_recordError` method is an internal implementation detail I think it's best to focus on the behavior change of the user-facing methods.